python/cudf/cudf/core/column/numerical_base.py

```diff
@@ -148,7 +109,18 @@ def quantile(
             )
         return result
 
-    def median(self, skipna: bool = None) -> NumericalBaseColumn:
+    def mean(self, dtype=np.float64, *args, **kwargs):
+        return self._reduce("mean", dtype=dtype, *args, **kwargs)
+
+    def var(self, dtype=np.float64, *args, **kwargs):
+        return self._reduce("var", dtype=dtype, *args, **kwargs)
+
+    def std(self, dtype=np.float64, *args, **kwargs):
+        return self._reduce("std", dtype=dtype, *args, **kwargs)
+
+    def median(
+        self, skipna: bool = None, *args, **kwargs
+    ) -> NumericalBaseColumn:
```

Comment:
Do we still need these?

Reply:
Unfortunately yes, we still need mean, var, and std because we need to override the default parameter for dtype. I omitted median because it has a significantly different implementation based on using quantile rather than being a direct calculation. Can we convert median to using the same machinery? That may be out of scope for this PR in any case, but may be worth tackling if we can do something cheaper than calling quantile.